deployment/ansible/playbook.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+
+  pre_tasks:
+    - name: Update APT cache
+      apt: update_cache=yes cache_valid_time=3600
+
+  roles:
+    - { role: "azavea.terraform" }
+    - { role: "azavea.java" } 
```

Comment:
Why is Java being installed? I believe that later on we'd move toward using a containerized OpenJDK.